### ISSUE TITLE
How can I use my own dataset to train the model?

### ISSUE THREAD

**Issue by wushilian** [wayaai/SimGAN#10]
I want to use the simgan to train other dataset，so what format does the data need？

**Comment by mjdietzx** [comment]
Uses [Keras image preprocessing](https://keras.io/preprocessing/image/). Look at keras documentation but this is pretty simple. Expects each dataset to be in a directory containing directories of images corresponding to label.

**Comment by dagoPA** [comment]
So in order to use a custom dataset what code file do i have to modify? Or it is just to have the right directory structure?

**Comment by mjdietzx** [comment]
Hey, you shouldn't have to modify the code. Synthesized images in one dir, refined in another. Make sure directories are named correctly. See keras image processing docs for more info.

**Comment by dagoPA** [comment]
Another question, Once the network is trained, how do i generate refined images?

**Comment by wushilian** [comment]
@dagoPA  Did you solve this problem?how to get refined images?

**Comment by dagoPA** [comment]
@wushilian I haven't yet Because i think this project only takes a random small set of data to inference and instead of saving it only saves the weights and the plot

**Comment by wushilian** [comment]
@dagoPA It seems like that.

**Comment by dagoPA** [comment]
@wushilian i think in the code i stead of predict batch we need a loop that iterates predicting each image in target directory

**Comment by wushilian** [comment]
@dagoPA that's all right.And I think cycle GAN and simGAN are very similar,so now I use cycleGAN to 
Converte synthetic data into real data

**Comment by dagoPA** [comment]
@wushilian I saw you forked that project, im new to github but i hope i can know your email

**Comment by wushilian** [comment]
@dagoPA wushilia@mail.ustc.edu.cn

**Comment by wushilian** [comment]
@dagoPA I guess you have saw my email,so I deleted it.
